infrastructure/keyvault.tf

```diff
@@ -0,0 +1,10 @@
+module "azurekeyvault" {
+  source                      = "git::https://github.com/hmcts/cnp-module-key-vault?ref=master"
+  name                        = "sds${var.product}-${var.environment}"
+  product                     = var.product
+  env                         = var.environment
+  managed_identity_object_ids = [azurerm_user_assigned_identity.managed_identity.principal_id]
```

Comment:
is there a reason you don't want to create inside the module? just to save time?